### PR TITLE
Feat/contactRecommendations: Contact slider on home page

### PR DESCRIPTION
**Pull request by askbk**

### CODE REVIEW

File: src/pages/home.jsx [hunk 1]
```diff
@@ -14,48 +14,68 @@ class Home extends React.Component {
...
 
     componentDidMount() {
-        fetch('http://localhost:5000/api/users?page=1')
-            .then(res => {
-                return res.json();
-            })
-            .then(users => {
-                this.setState({ data: users.data });
-            });
+        const userId = (JSON.parse(atob(localStorage.jwt.split(".")[1]))).sub,
```
Comment (98mux) [inline src/pages/home.jsx:28]: Trenger å håndtere når det ikke er noen token

File: src/pages/network.jsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ class Network extends React.Component {
     }
 
     componentDidMount() {
-        fetch('http://localhost:5000/api/users?page=2')
+        fetch('http://localhost/api/users?page=2')
```
Comment (edvardvb) [inline src/pages/network.jsx]: Why `page=2`?

File: src/pages/home.jsx [hunk 3]
```diff
@@ -14,48 +14,68 @@ class Home extends React.Component {
...
+            recommend = localStorage.userType === "employee" ? "jobseekers" : "employees";
+
+        fetch(`http://localhost/api/recommendations/${userId}/${recommend}`).then(res => {
+            return res.json();
+        }).then(users => {
+            this.setState({data: users});
+        });
+    }
+
+    handleContactRequestSent(id) {
```
Comment (edvardvb) [inline src/pages/home.jsx:45]: Not sure I understand this function

File: src/pages/network.jsx (hunk 2 above)
Comment (askbk) [reply src/pages/network.jsx]: Probably for testing with a dummy API. Will remove.

File: src/pages/home.jsx (hunk 3 above)
Comment (askbk) [reply src/pages/home.jsx:45]: If a contact request was successfully sent from the home page, the SwiperSlide component needs to be removed in order to not leave a gap when the Match component is hidden. This function accomplishes that by removing the user from the array of contact recommendations.

Will add clarifying comments.

**Review by edvardvb — APPROVED** [review]

**Pull request merged**